general/multi_companies/manage_multi_companies.rst

```diff
@@ -0,0 +1,134 @@
+Multi Companies
+===============
+
+A centralized management environment allows you to select multiple companies simultaneously, and set
+their specific warehouses, customers, equipment and contacts. It provides you the ability to
+generate reports of aggregated figures without the need of switching interfaces, which facilitates
+daily tasks and the overall management process.
+
+Manage companies and records
+----------------------------
+
+Go to :menuselection:`Settings --> Manage Companies` and fill in the form with your company’s
+information. If a *Parent Company* is selected, records are shared between the two companies.
```

Comment:
```suggestion
information. If a *Parent Company* is selected, records are shared between the two companies.
```

Not sure, but I think that records from the child company are accessible by the parent users but not the opposite, right?
(Or it's completely the opposite haha).

If I'm right, I think that it should be said here :)

Reply:
Good one. So, records are accessible from both environments as long as both environments are active. Example: a record created in a "child" company can be seen by the parent, and vice-versa, as long as both dbs are activated.

:) thanks!

Reply:
Thanks for the explanation! :+1: